examples/opensora_hpcai/opensora/models/layers/blocks.py

```diff
@@ -416,13 +417,15 @@ def construct(
         T: Optional[int] = None,
         S: Optional[int] = None,
     ) -> Tensor:
-        T = T or self.d_t
-        S = S or self.d_s
-        shift, scale = (self.scale_shift_table[None] + t[:, None]).chunk(2, axis=1)
+        if T is None:
+            T = self.d_t
+        if S is None:
+            S = self.d_s
+        shift, scale = mint.chunk(self.scale_shift_table[None] + t[:, None], 2, 1)
         x = t2i_modulate(self.norm_final(x), shift, scale)
 
-        if frames_mask is not None:
-            shift_zero, scale_zero = (self.scale_shift_table[None] + t0[:, None]).chunk(2, axis=1)
+        if True:
```

Comment:
```suggestion
        if self.has_frames_mask:
```
add `self.has_frames_mask = True`  in __init__ when v1.0